cmd/sovereignnode/systemTestDemo/mockNotifier/vars.go

```diff
@@ -10,6 +10,8 @@ import (
 const (
 	addressLen = 32
 	hashSize   = 32
+	lenSize    = 4
```

Comment:
Len of what?

Reply:
For coding, first you encode the length, the you encode the item. So it's length of the item

Reply:
Yes, but please rename it :D